eternalcore-core/src/main/java/com/eternalcode/core/feature/seen/SeenCommand.java

```diff
@@ -0,0 +1,70 @@
+package com.eternalcode.core.feature.seen;
+
+import com.eternalcode.annotations.scan.command.DescriptionDocs;
+import com.eternalcode.core.injector.annotations.Inject;
+import com.eternalcode.core.notice.NoticeService;
+import com.eternalcode.core.user.User;
+import com.eternalcode.core.util.DurationUtil;
+import dev.rollczi.litecommands.annotations.argument.Arg;
+import dev.rollczi.litecommands.annotations.command.Command;
+import dev.rollczi.litecommands.annotations.context.Context;
+import dev.rollczi.litecommands.annotations.execute.Execute;
+import dev.rollczi.litecommands.annotations.permission.Permission;
+import org.bukkit.OfflinePlayer;
+import org.bukkit.Server;
+
+import java.time.Duration;
+import java.time.Instant;
+
+@Command(name = "seen", aliases = { "lastonline" })
+@Permission("eternalcore.seen")
+class SeenCommand {
+
+    private final Server server;
+    private final NoticeService noticeService;
+
+    @Inject
+    public SeenCommand(Server server, NoticeService noticeService) {
+        this.server = server;
+        this.noticeService = noticeService;
+    }
+
+    @Execute
+    @DescriptionDocs(description = "Shows when the player was last seen on the server")
+    void execute(@Context User sender, @Arg User target) {
```

Comment:
it is possible to make it also for console?

![image](https://github.com/user-attachments/assets/75e4e56d-6c9c-4856-969a-5f9b2ce7c8c2)